npx tsc fails due to absolute import in SDK published types

**Describe the bug**
Running `npx tsc` on plugins that use bigbluebutton-html-plugin-sdk fails with:
```
node_modules/bigbluebutton-html-plugin-sdk/dist/cjs/core/auxiliary/plugin-information/locale-messages/types.d.ts:1:27 - error TS2307: Cannot find module 'src/core/api/types' or its corresponding type declarations.

1 import { PluginApi } from 'src/core/api/types';
```
The error is caused by an absolute import ('src/...) in the published .d.ts file. (make sure `skipLibCheck` is `false` in `.tsconfig`)

Happens in v0.1.5^ and v0.0.79^

- https://github.com/bigbluebutton/bigbluebutton-html-plugin-sdk/commit/b4dfc0437f2da3bbf6f2301a1c028aa8063d6a52

--- a/src/core/auxiliary/plugin-information/locale-messages/types.ts
+++ b/src/core/auxiliary/plugin-information/locale-messages/types.ts
@@ -1,4 +1,4 @@
-import { PluginApi } from 'src/core/api/types';
+import { PluginApi } from '../../../api/types';
 
 interface UseLocaleMessagesProps {
   pluginApi: PluginApi;

